packages/edit-site/src/components/block-editor/use-site-editor-settings.js

```diff
@@ -127,75 +115,46 @@ export default function useSiteEditorSettings() {
 			usedPostId
 		);
 		return {
-			restBlockPatterns: select( coreStore ).getBlockPatterns(),
-			restBlockPatternCategories:
-				select( coreStore ).getBlockPatternCategories(),
 			templateSlug: _record.slug,
-			userPatternCategories: getUserPatternCategories(),
+			focusMode: !! __unstableGetPreference( 'focusMode' ),
+			isDistractionFree: !! __unstableGetPreference( 'distractionFree' ),
+			hasFixedToolbar: !! __unstableGetPreference( 'fixedToolbar' ),
+			keepCaretInsideBlock: !! __unstableGetPreference(
+				'keepCaretInsideBlock'
+			),
```

Comment:
I think we can now select from the preferences store directly instead of using `__unstableGetPreference`.

With the changes in this PR we could potentially remove `__unstableGetPreference` completely, I think I only introduced it because it was the only way to incorporate preference store values into the memoized settings selector at the time.

I'd be happy to follow up on this if you don't want to incorporate it into this PR right now 👍